### PR TITLE
Update README for Prometheus Client Output

### PR DESCRIPTION
**Pull request by trastle**
Update the README of the Prometheus Output with the example configuration.

### CODE REVIEW

File: plugins/outputs/prometheus_client/README.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Prometheus Client Service Output Plugin
 
-This plugin starts a Prometheus Client, listening on a port defined in the
-configuration file.
+This plugin starts a [Prometheus](https://prometheus.io/) Client, it exposes all metrics on `/metrics` to be polled by a Prometheus server.
```
Comment (sparrc) [inline plugins/outputs/prometheus_client/README.md]: why did you reword this?

File: plugins/outputs/prometheus_client/README.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
 
-It exposes all metrics on `/metrics` to be polled by a Prometheus server.
+## Configuration
+
+    # Publish all metrics to /metrics for Prometheus to scrape
+	[[outputs.prometheus_client]]
```
Comment (sparrc) [inline plugins/outputs/prometheus_client/README.md]: this should be in an unindented code block, ie, with "\`\`\`" on either side. See other plugin readmes for an example.

File: plugins/outputs/prometheus_client/README.md (hunk 1 above)
Comment (trastle) [inline plugins/outputs/prometheus_client/README.md]: The original wording made more sense to me when the port was the only configuration option. When the following block shows the example config it seems odd to specifically call out the port here. 

Additionally the config was referred to as port here and address within the example config.

File: plugins/outputs/prometheus_client/README.md (hunk 2 above)
Comment (trastle) [inline plugins/outputs/prometheus_client/README.md]: Will do.

**Comment by trastle** [comment]
@sparrc The other thing I was considering was listing the required and optional parameters. Similar to: https://github.com/influxdata/telegraf/tree/master/plugins/outputs/kafka but this felt like overkill. Thoughts?

**Comment by sparrc** [comment]
yes, probably overkill

File: plugins/outputs/prometheus_client/README.md (hunk 1 above)
Comment (sparrc) [inline plugins/outputs/prometheus_client/README.md]: OK, fair enough 👍

**Comment by danielnelson** [comment]
@trastle are you still planning to update or would you like me to finish this up?

**Comment by trastle** [comment]
@danielnelson sorry completely forgot about this. @sparrc I have made the requested formatting changes. Sorry for the delay.

**Pull request merged**